projects/experimental/directives/surface/surface.style.less

```diff
@@ -164,3 +164,20 @@ input[tuiSurfaceLayer] {
         }
     }
 }
+
+[tuiSurface][data-surface='dark'] {
+    background: linear-gradient(90deg, rgba(125, 140, 160, 1) 0%, rgba(100, 115, 130, 1) 31%);
```

Comment:
This looks wrong. It probably should go top to bottom and on 31% but full height. Did you check it with designers?

Reply:
fixed